src/main/java/org/pac4j/sparkjava/RequiresAuthenticationFilter.java

```diff
@@ -65,6 +66,10 @@ public void handle(Request request, Response response) {
 			Client<Credentials, CommonProfile> client = clients.findClient(this.clientName);
             try {
                 client.redirect(context, true, false);
+                // fix for SAML redirection
+                if (context.getStatus() == 200) {
```

Comment:
When the response has a 200 status, it is overridden by the view. So halt in that case.